### ISSUE TITLE
Plone 5.2rc1: "No module named pip" on Ubuntu 18.04.2 LTS

### ISSUE THREAD

**Issue by tkimnguyen** [plone/Installers-UnifiedInstaller#83]
```
root@2019:/root/Plone-5.2rc1-UnifiedInstaller# ./install.sh --target=/usr/local/plone-5.2rc1 zeo

Testing /usr/bin/python2.7 for Zope/Plone requirements....
/usr/bin/python2.7 looks OK. We will use it.


Root install method chosen. Will install for use by users:
  ZEO & Client Daemons:      plone_daemon
  Code Resources & buildout: plone_buildout


Detailed installation log being written to /root/Plone-5.2rc1-UnifiedInstaller/install.log
Installing Plone 5.2rc1 at /usr/local/plone-5.2rc1

Using useradd and groupadd to create users and groups.
plone_group already exists; no need to create it.
User plone_daemon already exists. No need to create it.
Adding user plone_daemon to group plone_group.
User plone_buildout already exists. No need to create it.
Adding user plone_buildout to group plone_group.
Creating Python virtual environment.
New python executable in /usr/local/plone-5.2rc1/zeocluster/bin/python2.7
Also creating executable in /usr/local/plone-5.2rc1/zeocluster/bin/python
Command /usr/local/plone-5.2...luster/bin/python2.7 -m pip config list had error code 1
Installing setuptools, pip, wheel...

  Complete output from command /usr/local/plone-5.2...luster/bin/python2.7 - setuptools pip wheel:
  Traceback (most recent call last):
  File "<stdin>", line 13, in <module>
ImportError: No module named pip
----------------------------------------
...Installing setuptools, pip, wheel...done.
Installing Python requirements in virtual environment.
Unable to install Python requirements in virtual environment.

Installation has failed.
See the detailed installation log at /root/Plone-5.2rc1-UnifiedInstaller/install.log
to determine the cause.
```

The detailed installation log:

```
Detailed installation log
Starting at Sun Mar 17 13:16:00 UTC 2019
Traceback (most recent call last):
  File "virtualenv.py", line 2566, in <module>
    main()
  File "virtualenv.py", line 810, in main
    symlink=options.symlink,
  File "virtualenv.py", line 1105, in create_environment
    install_wheel(to_install, py_executable, search_dirs, download=download)
  File "virtualenv.py", line 952, in install_wheel
    _install_wheel_with_search_dir(download, project_names, py_executable, search_dirs)
  File "virtualenv.py", line 1042, in _install_wheel_with_search_dir
    call_subprocess(cmd, show_stdout=False, extra_env=env, stdin=script)
  File "virtualenv.py", line 903, in call_subprocess
    raise OSError("Command {} failed with error code {}".format(cmd_desc, proc.returncode))
OSError: Command /usr/local/plone-5.2...luster/bin/python2.7 - setuptools pip wheel failed with error code 1
sudo: /usr/local/plone-5.2rc1/zeocluster/bin/pip: command not found
```

**Comment by tkimnguyen** [comment]
Weirdly (?) this 5.2rc1 unified installer runs fine on macOS 10.14.3

**Comment by smcmahon** [comment]
This is resolved with the 5.2.0-r2 installer.
